docs/user/supported_devices.rst

```diff
@@ -206,6 +206,10 @@ ath79-generic
 
   - Archer C6 (v2)
 
+* Ubiquity
+
+  - Nanostation AC (WA)
```

Comment:
I would just call it "Nanostation AC", which is how Ubiquiti itself calls the device: https://www.ui.com/airmax/nanostation-ac/

Reply:
The device needs to be flagged as broken, so it won't appear on the supported devices list at all.

Reply:
Nope, It's just not supported per definition if it is broken. So no need for a footnote.

Reply:
Wouldn't this mean I need to remove the whole change to this file?